scribble-doc/scribblings/scribble/manual.scrbl

```diff
@@ -1379,6 +1392,11 @@ If @racket[#:value value-expr-datum] is given, @racket[value-expr-datum]
 is typeset using @racket[racketblock0] and included in the documentation.
 Wide values are put on a separate line.
 
+@racket[#:auto-value] is similar to @racket[#:value value-expr-datum],
```

Comment:
Nit: I'd rephrase to avoid starting the sentence with a lowercase keyword, maybe "Supplying #:auto-value ...".